internal/convert.go

```diff
@@ -169,16 +169,36 @@ func (conv *Conv) SetDataMode() {
 }
 
 // GetDDL Schema returns the Spanner schema that has been constructed so far.
-// Return DDL in alphabetical table order.
+// We sort DDL in alphabetical order, but we maintain the order of interleaved
```

Comment:
Suggest revising comment so that it reflects functionality rather than implementation.  Maybe something like:

// GetDDL Schema returns the current Spanner schema that has been constructed so far.
// We return DDL in alphabetical order with one exception: interleaved tables are
// potentially out of order since they must appear after the definition of their
// parent table.

Reply:
added comment